Orange/widgets/data/owtranspose.py

```diff
@@ -92,6 +95,10 @@ def set_data(self, data):
         if self.feature_model:
             self.openContext(data)
         self.unconditional_apply()
+        if data:
```

Comment:
I'd rather this happens before `self.apply()`. The apply could take a while with a big dataset. In that case the info is wrongly set through the whole procedure.

The best place is probably right after `self.data = data` line.